zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/exporter/ExporterEnableTest.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.exporter;
+
+import static io.camunda.zeebe.test.StableValuePredicate.hasStableValue;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.broker.shared.BrokerConfiguration.BrokerProperties;
+import io.camunda.zeebe.broker.system.configuration.ExporterCfg;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.exporter.api.Exporter;
+import io.camunda.zeebe.exporter.api.context.Context;
+import io.camunda.zeebe.exporter.api.context.Controller;
+import io.camunda.zeebe.management.cluster.PlannedOperationsResponse;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.qa.util.actuator.ExportersActuator;
+import io.camunda.zeebe.qa.util.cluster.TestCluster;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.qa.util.topology.ClusterActuatorAssert;
+import io.camunda.zeebe.test.util.asserts.TopologyAssert;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources.AutoCloseResource;
+import java.nio.ByteBuffer;
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.stream.IntStream;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+
+@Timeout(2 * 60) // 2 minutes
+@ZeebeIntegration
+@AutoCloseResources
+final class ExporterEnableTest {
+  private static final int PARTITIONS_COUNT = 2;
+  private static final String EXPORTER_ID_1 = "exporter-1";
+  private static final String EXPORTER_ID_2 = "exporter-2";
+
+  @TestZeebe
+  private final TestCluster cluster =
+      TestCluster.builder()
+          .withBrokersCount(3)
+          .withPartitionsCount(PARTITIONS_COUNT)
+          .withReplicationFactor(3)
+          .withEmbeddedGateway(false)
+          // We have to restart brokers in the test. So use standalone gateway to avoid potentially
+          // accessing an unavailable broker
+          .withGatewaysCount(1)
+          .withBrokerConfig(b -> withCustomerExporters(b.brokerConfig()))
```

Comment:
:thought_balloon: `TestStandaloneBroker` also has a method called `withExporter` where you can configure you exporters, e.g.

```java
.withBrokerConfig(
  b -> 
    b.withExporter(EXPORTER_ID_1, config -> config.setClassName(TestExporter.class.getName()))
     .withExporter(EXPORTER_ID_2, config -> config.setClassName(TestExporter.class.getName())));
```

Reply:
TIL :+1: